test/lib/meetup_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+require_relative '../support/meetups/sample_api_responses'
+
+class MeetupTest < ActiveSupport::TestCase
+  test "some test" do
+    Meetup.stubs(:get).with("/pro/operationcode/groups", options).returns(group_endpoint_response)
+
+    parsed_response = Meetup.new.operationcode_data
```

Comment:
So since `Meetup.new.operationcode_data` calls `self.class.get` inside of it, it respects that stub you set up.